Pod/Classes/URBNDataSourceAdapter.m

```diff
@@ -276,7 +276,7 @@ - (void)registerCellClass:(Class)cellClass withIdentifier:(NSString *)identifier
     ASSERT_TRUE(self.tableView || self.collectionView);
     
     identifier = identifier?:NSStringFromClass(cellClass);
-    UINib* nib = [self nibWithName:identifier];
+    UINib* nib = [self nibWithName:NSStringFromClass(cellClass)];
```

Comment:
So we're saying.   Given an identifier and given an identifier and given a cellClass always use the cellClass?

What if the nib is named based off the identifier?